src/index.html

```diff
@@ -17,6 +17,86 @@
     />
   </head>
   <body>
-    <h1>Moyo header</h1>
+    <header>
+      <a
+        href="#"
+        class="img"
+      >
+        <img
+          src="images/logo.png"
+          alt="Moyo__Img"
+        />
+      </a>
+
+      <nav>
+        <ul class="ul__section">
+          <li class="nav__section">
+            <a
+              href="#"
+              class="nav__button is-active first"
```

Comment:
Don't use spaces in `<a>` tag's `href` property. Anchor links should start with the `#` symbol followed by the target element's ID. For example, `href="#apple"`.